tableaudocumentapi/datasource.py

```diff
@@ -63,7 +66,15 @@ def _column_object_from_metadata_xml(metadata_xml):
     return _ColumnObjectReturnTuple(field_object.id, field_object)
 
 
+def make_unique_name(dbclass):
+    rand_part = ''.join(random.choice(
```

Comment:
Our unique names are a hash based on the current time, so we should probably do that rather than random data.  I can find the relevant code if you need it.